.github/workflows/coverage.yml

```diff
@@ -37,6 +37,8 @@ jobs:
           api-level: ${{ matrix.api-level }}
           arch: x86_64
           ndk: 21.4.7075529
+          sdcard-path-or-size: '1000M'
+          disable-animations: false
```

Comment:
Is it necessary here?

Reply:
Test on android 21 , search fragment is always failing on CI while working fine with locally by enabling the animation and placing this sdcard config make CI to work so its required for test.